src/pybind/mgr/dashboard/frontend/cypress/integration/orchestrator/workflow/02-create-cluster-add-host.feature

```diff
@@ -0,0 +1,41 @@
+@login
+Feature: Cluster expansion host addition
+
+    Add some hosts and perform some host related actions like editing the labels
+    and removing the hosts from the cluster and verify all of the actions are performed
+    as expected
+
+    Background: Cluster expansion wizard
+        Given the welcome screen
+        Then go to 'wizard'
+        Then go to 'Add Host' section
```

Comment:
```suggestion
        Given (the|a)? (I|user|operator|she|he|they) ((am|is) in|go(es)? to) the welcome screen
        And she is in the 'wizard' # wizard is a page?
        And she is to 'Add Host' section
```

Keep in mind that BDD, Gherking, defining features in a natural language is about making features and tests understandable for non-expert users. By skipping the subject in these, we may lose the focus on the fact that this is user-centric (BDD is mostly used for User Acceptance Testing), so we should highlight who the user is. And additionally, in future features there might be multiple users (admin, guest, etc).

We don't have to match for specific strings, but just put a placeholder in the subject. I've even seen [an example where they created a pattern to match lots of possible subjects](https://testzius.wordpress.com/2017/01/30/supercharging-your-step-definitions/).

Additionally, Background will mostly be about Given clauses, not When/Then, since Background is setting the scene in a declarative fashion, but not imperatively stating any steps and expected outcomes (that's for scenarios, which btw are now and IMHO better called `Examples` as well).